format_notification.py

```diff
@@ -0,0 +1,72 @@
+#!/bin/env python
+
+# this while loop is for not considering two string fields which you get right after starting a process
+# Test string ->['string "ShanonPark posted in "HEY! YOU! What are you up to right now! (Part 3)" - Linkin Park / LPU Forums"'] 2 
+
+import time
+import sys
+import notify2
+from setup import main
+
+def EditString(data: str) -> str:
+    return data[0].split('"', 1)[1].replace('"', '')
+
+def Notify(summary, data, urgency, timeout=800000):
+    notify2.init("AutoReplyToLP")
+    pop_up = notify2.Notification(summary, data)
+    pop_up.set_urgency(urgency)
+    pop_up.set_timeout(timeout)
+    return pop_up
+
+def countdown_timer(timeMS):
+    # timeMS should be in millisecond
+    minute = timeMS // 60000
+    second = 59
```

Comment:
59 doesn't seem to be a second to me?

Reply:
oh.. this is similar to `minute :  second` -> `14:59`
Now the counter starts from `14:59` and will get reduce to `0:0`

I guess, you said something related to use time module, right? 
where is it, you deleted or what?